models/contact.go

```diff
@@ -25,6 +25,18 @@ type Contact struct {
 	Status response.ContactStatus `json:"status" example:"unconfirmed"`
 }
 
+type ContactConfirmationData struct {
+	// XPubID is a xpub id related to contact.
+	XPubID string `json:"xpubId" example:"68af358bde7d8641621c7dd3de1a276c9a62cfa9e2d0740494519f1ba61e2f4a"`
+	// Paymail is a paymail address related to contact.
+	Paymail string `json:"paymail" example:"test@test.test"`
+}
+
+type AdminConfirmContactPair struct {
+	ContactA ContactConfirmationData `json:"contactA"`
```

Comment:
As we talked,
maybe it would be much better to provide only the pair of paymail addresses. And the endpoint would check if ContacA has ContactB and if ContactB has ContactA and then and only then it will confirm this pair.